src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -508,7 +510,11 @@ public function make_subsize( $size_data ) {
 		if ( is_wp_error( $resized ) ) {
 			$saved = $resized;
 		} else {
-			$saved = $this->_save( $this->image );
+			if ( $mime_type ) {
+				$saved = $this->_save( $this->image, null, $mime_type );
+			} else {
+				$saved = $this->_save( $this->image );
+			}
```

Comment:
See above, this can be simplified:
```suggestion
			$saved = $this->_save( $this->image, null, $mime_type );
```